pastis/matrix_generation/matrix_building_numerical.py

```diff
@@ -458,15 +458,15 @@ def _hicat_matrix_one_pair(norm, wfe_aber, resDir, savepsfs, saveopds, segment_p
     return contrast, segment_pair
 
 
-def _rst_matrix_one_pair(norm, wfe_aber, resDir, savepsfs, saveopds, actuator_pair):
+def _rst_matrix_one_pair(norm, dh_mask, wfe_aber, resDir, savepsfs, saveopds, actuator_pair):
     """
     Function to calculate RST mean contrast of one DM actuator pair in CGI.
     :param norm: float, direct PSF normalization factor (peak pixel of direct PSF)
     :param wfe_aber: calibration aberration per segment in m
     :param resDir: str, directory for matrix calculations
     :param savepsfs: bool, if True, all PSFs will be saved to disk individually, as fits files
     :param saveopds: bool, if True, all pupil surface maps of aberrated segment pairs will be saved to disk as PDF
-    :param actuator_pair:
+    :param actuator_pair: pair of actuator where is injected aberration
```

Comment:
It is always good to specify the type in the docstring, e.g.: `tuple, pair of actuators to aberrate. If same segment gets passed in both tuple entries, the actuator will be aberrated only once.`